libs/coin-modules/coin-filecoin/src/api/api.ts

```diff
@@ -19,62 +18,68 @@ import {
   ERC20BalanceResponse,
 } from "../types";
 import { FilecoinFeeEstimationFailed } from "../errors";
+import { LiveNetworkRequest } from "@ledgerhq/live-network/network";
 
-const getFilecoinURL = (path?: string): string => {
+const txsPerPageLimit = 1000;
+const currentVersion = "/v2";
+const fromHeightQueryParam = "from_height";
+
+const getFilecoinURL = (version?: string, path?: string): string => {
   const baseUrl = getEnv("API_FILECOIN_ENDPOINT");
   if (!baseUrl) throw new Error("API base URL not available");
 
-  return `${baseUrl}${path ? path : ""}`;
+  return `${baseUrl}${version ? version : ""}${path ? path : ""}`;
 };
 
-const fetch = async <T>(path: string) => {
-  const url = getFilecoinURL(path);
+const fetch = async <T>(path: string, version: string) => {
```

Comment:
this doesn't match the suggestion i made on previous comment https://github.com/LedgerHQ/ledger-live/pull/12304#discussion_r2477249853

and call signature should be IMO
```
type FetchProps = {
   version?: string
}
const fetch = async <T>(path: string, {version}:FetchProps = {version:currentVersion }) => {
   ...
}

const pathData = await fetch('some/path')
```

Reply:
Applied this, thanks!